cpp-client/README.md

```diff
@@ -57,10 +59,9 @@ C++ compiler and tool suite (cmake etc).
    ```
    cd $DHSRC/deephaven-core/cpp-client/deephaven/
    mkdir build && cd build
-   export DEEPHAVEN_LOCAL=$HOME/dhcpp/local  # This should reflect your selection in the previous point.
-   export CMAKE_PREFIX_PATH=${DEEPHAVEN_LOCAL}/abseil:${DEEPHAVEN_LOCAL}/boost:${DEEPHAVEN_LOCAL}/cares:${DEEPHAVEN_LOCAL}/flatbuffers:${DEEPHAVEN_LOCAL}/gflags:${DEEPHAVEN_LOCAL}/immer:${DEEPHAVEN_LOCAL}/protobuf:${DEEPHAVEN_LOCAL}/re2:${DEEPHAVEN_LOCAL}/zlib:${DEEPHAVEN_LOCAL}/grpc:${DEEPHAVEN_LOCAL}/arrow:${DEEPHAVEN_LOCAL}/deephaven
-   export NCPUS=$(getconf _NPROCESSORS_ONLN)
-   cmake -DCMAKE_INSTALL_PREFIX=${DEEPHAVEN_LOCAL}/deephaven .. && make -j$NCPUS install
+   export DHCPP=$HOME/dhcpp  # This should reflect your selection in the previous point.
```

Comment:
Do we need / want to repeat this?  This is the same export DHCPP command that we did above in line 47.